person.rb

```diff
@@ -1,7 +1,7 @@
 require './nameable'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rental
```

Comment:
- [ ] As the previous reviewer mentioned kindly use `attr_accessor` for the `:rental`, so you can access it and modify it also